nuage-openshift-ansible/roles/nuage_master/tasks/serviceaccount.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Create Admin Service Account
+  oc_serviceaccount:
```

Comment:
This module does not exist in ansible, it only exists in the openshift-ansible project, same for all other `oc_*` modules you use in the below items.

These modules are all defined here: https://github.com/openshift/openshift-ansible/tree/master/roles/lib_openshift/library . 

This means that we need to: 
1. Get the Openshift Ansible repo checked out for the proper tag
2. Put our files in the appropriate location (roles in the roles folder, playbooks in the root folder, etc)
3. Add the following to the meta of any roles using these special `oc_` modules:
```
---
dependencies:
- role: lib_openshift
- role: lib_utils
```

Resulting in a non-optimal decoupling, however, this can be scripted/automated as well where we provide a generic script that:
- Clones the openshift repo
- Checks out the right tag
- Moves our files in the right places (instead of doing a patch, patches are dirty).

Another solution would be to create a nuage-openshift-ansible repo, where we add our roles and then work with Red Hat so they include us as a submodule in their github repo... But that's always a bit tougher to get done (and needs a proper mechanism to reference the right tag in our repo for each of their releases, which will be harder to maintain).